projects/angular/src/forms/datepicker/day.ts

```diff
@@ -27,6 +27,7 @@ import { DateNavigationService } from './providers/date-navigation.service';
       (click)="selectDay()"
       (focus)="onDayViewFocus()"
       [attr.aria-label]="dayString"
+      [attr.aria-current]="dayView.isTodaysDate"
```

Comment:
This isn't quite right because the value should be literally "date" as shown in the graphic. But this is actually unrelated to I indicating the selected day, so I think this should removed. I'll open a separate PR.

Reply:
This is in #253.